packages/@ember/-internals/glimmer/lib/modifiers/custom.ts

```diff
@@ -9,11 +11,24 @@ export interface CustomModifierDefinitionState<ModifierInstance> {
   delegate: ModifierManagerDelegate<ModifierInstance>;
 }
 
-export interface Capabilities {}
+export interface OptionalCapabilities {
+  disableLifecycleTracking?: boolean;
+}
+
+export interface Capabilities {
+  disableLifecycleTracking: boolean;
+}
 
 // Currently there are no capabilities for modifiers
-export function capabilities(_managerAPI: string, _optionalFeatures?: {}): Capabilities {
-  return {};
+export function capabilities(
+  managerAPI: '3.13',
+  optionalFeatures: OptionalCapabilities
+): Capabilities {
+  assert('Invalid modifier manager compatibility specified', managerAPI === '3.13');
+
+  return {
+    disableLifecycleTracking: Boolean(optionalFeatures.disableLifecycleTracking),
```

Comment:
This name does not match up at all with "auto tracking" to me, "disable lifecycle tracking" in the abstract seems like it would be more about handling teardown?

Some suggestions:

* `disableTracking`
* `disableTrackedPropertyTracking`
* `disableAutoTracking`